stories/index.js

```diff
@@ -211,4 +211,10 @@ storiesOf("Marketing Thumb", module)
       text="hundreds of fishing experiences around the world handpicked by Fishtripr."
       />`
   );
-
+storiesOf("Messages", module)
```

Comment:
You can now create the story on a separate file like the others new one that we made together. It will make more clarity.
eg: `conversation/receiver-header/index.js`